.travis.yml

```diff
@@ -63,7 +63,7 @@ jobs:
       if: commit_message ~= /\[trigger release\]/ AND branch ~= /^(master|SP\/.+|HF\/.+)$/
       before_install: bash _ci/init.sh
       before_script: travis_wait bash _ci/cache_artifacts.sh
-      script: travis_wait 30 bash _ci/release.sh
+      script: travis_wait 45 bash _ci/release.sh
```

Comment:
I would go even higher just in case, maybe an hour

Reply:
OK, I'll bump to 55 and we can come back down later, if possible.